doc/manual/citing_aspect.bib

```diff
@@ -1096,7 +1096,7 @@ @Article{geosciences12020050
 DOI = {10.3390/geosciences12020050}
 }
 
-article{doi:10.1126/sciadv.abg1626,
+@article{doi:10.1126/sciadv.abg1626,
```

Comment:
I'm already fixing this in #4510.

Reply:
Thanks @bangerth. Looks like I did introduce a new error of a different kind. I will look out for this in the future.

Reply:
Can you just remove that one character again? It currently causes a merge conflict.